src/CSharp/EasyMicroservices.StorageMicroservice.Domain/Contracts/Requests/FolderRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EasyMicroservices.StorageMicroservice.Contracts
+{
+    public class AddFolderDom
```

Comment:
Dont use the 'Dom' suffix, only Contract is good.
move it to requests.
rename it to AddFolderRequestContract